packages/orbit-components/src/Layout/LayoutColumn/index.tsx

```diff
@@ -4,13 +4,30 @@ import * as React from "react";
 import styled, { css } from "styled-components";
 
 import defaultTheme from "../../defaultTheme";
-import getViewportHideStyles from "../../Hide/helpers/getViewportHideStyles";
```

Comment:
Wouldn't it be better to keep this function for now until the Layout component is migrated?

Reply:
I found that better to remove it and just directly past it inside the Layout. It would make sense to keep inside Stack, if it were used on multiple places

Reply:
Ok, I see, not really strong opinion on this.